corehq/apps/user_importer/importer.py

```diff
@@ -311,6 +315,8 @@ def create_or_update_web_user_invite(email, domain, role_qualified_id, upload_us
     )
     if invite_created and send_email:
         invite.send_activation_email()
+    if invite_created and user_change_logger:
```

Comment:
when would `user_change_logger` be None?

Reply:
https://github.com/dimagi/commcare-hq/pull/29935/commits/e8736f5060bb376fd3427682e5f662ea7672f8a7#r656096471